code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -2747,10 +2747,10 @@
 	name = "CHUNK box"
 	desc = "A bar of \"The <b>CHUNK</b>\" brand chocolate. <i>\"The densest chocolate permitted to exist according to federal law. We are legally required to ask you not to use this blunt object for anything other than nutrition.\"</i>"
 	icon_state = "chunk"
-	force = 10 //LEGAL LIMIT OF CHOCOLATE
+	force = 35 //LEGAL LIMIT OF CHOCOLATE
 	bitesize = 3
 	wrapper = /obj/item/trash/chunk
-	list_reagents = list("nutriment" = 5, "coco" = 10)
+	list_reagents = list("nutriment" = 10, "coco" = 10)
```

Comment:
This actually makes it better _the_ nutriment choice to supply the marines. Either increase the size or remove this buff.

Reply:
aye aye captain. i figured more chocolate would = more nutriment, but i'll change it